packages/publish-spec-to-team/src/index.ts

```diff
@@ -0,0 +1,129 @@
+import { Command } from "commander";
+import fetch, { fileFrom, FormData } from "node-fetch";
+import path from "path";
+import esbuild from "esbuild";
+import { createFile, exec, validateSpecName } from "./utils";
+import { API_BASE, cleanTempDir, tempDir } from "./constants";
+// @ts-ignore package.json is always in '../' as long as we build src/ to lib/
+import packageJSON from "../package.json";
+import { BuildError, GenericErrorEnum, PublishError, ValidationError } from "./errors";
+
+export interface RunOptions {
+  name?: string;
+  namespace?: string;
+  token: string;
+  specPath?: string;
+  binaryPath?: string;
+  subcommandName?: string;
+  framework?: string;
+}
+
+const program = new Command()
+  .name(packageJSON.name)
+  .version(packageJSON.version)
+  .description("Publish a spec to fig teams");
+
+program
+  .requiredOption("-t, --token <string>", "A fig token")
+  .option("-c, --namespace", "Set the namespace of the published spec")
```

Comment:
I think it will be helpful to call this `--team <team>` and then if it is omitted, the namespace defaults to the user.